src/components/MegadraftEditor.js

```diff
@@ -162,11 +181,7 @@ export default class MegadraftEditor extends Component {
           className="megadraft-editor"
           id="megadraft-editor"
           ref="editor">
-          <Sidebar
-            plugins={plugins}
-            editorState={editorState}
-            readOnly={this.state.readOnly}
-            onChange={this.onChange} />
+          {this.Sidebar}
```

Comment:
I believe that this is a desirable behavior, the `render` method should have control over the sidebar and React does the rest.

We could run into problems of not updating the sidebar.

If it gets slow we'll have to deal with later.